### ISSUE TITLE
Integration with Gradle

### ISSUE THREAD

**Issue by chakkasm** [sksamuel/scapegoat#195]
I am trying to integrate with my gradle build and included the following in build.gradle
 compile(group: 'io.jenner', name: 'gradle-scapegoat', version: '0.90.0')

I don't see tasks related to running scapegoat. Am I missing something? Or let me know how to run scapegoat using gradle and where to find output (reports)

**Comment by willis7** [comment]
That will only provide you with the dependency, I wont run/create a scapegoat task. Once I find the solution I will add it here.

**Comment by BalmungSan** [comment]
Hi @jackandjackadswithsm and  @willis7, did you managed to make it work?

**Comment by carlspring** [comment]
Hi!

I was wondering, if there was any update on this? Are there any examples of how to get `scapegoat` working in Gradle?

Thanks! :)

**Comment by Axblade** [comment]
It turns out it's not that different from Maven-Scapegoat integration.
I managed to get it working with basic scala library gradle project (generated with `gradle init`).
The resulting build file is https://gist.github.com/Axblade/6d637290c9c178bbd8d156c69a51ff43

@jackandjackadswithsm , @willis7 , @BalmungSan , @carlspring could you check if it works for you? If it does, I'll provide an update for the README

**Comment by willis7** [comment]
@Axblade unfortunately, I'm no longer working with the client or Scala. However, from what you have it looks plausible. It also suggests this runs with every compile? Is that correct?

**Comment by Axblade** [comment]
@willis7 yep, it run on every compile attempt. I guess the same happens with maven. 
Chances are, better integration requires a full-featured plugin.

**Comment by carlspring** [comment]
@eugene-sy ,

Your gist seems to no longer exist! Would you mind re-sharing it?

**Comment by eugene-sy** [comment]
@carlspring Ah, sorry, it was an unforeseen side effect of GitHub handle change
Here is the updated link https://gist.github.com/eugene-sy/6d637290c9c178bbd8d156c69a51ff43

**Comment by carlspring** [comment]
@eugene-sy ,

Thanks a lot! I'll have a look! :)

**Comment by eugene-sy** [comment]
I'm currently working on a Gradle plugin for Scapegoat.
It would be nice if someone could help me validating it and suggest missing/desired functionality. Obviously code contributions are welcome and encouraged.
Here is the repo: https://github.com/eugene-sy/gradle-scapegoat-plugin
And configurations examples are here: https://github.com/eugene-sy/gradle-scapegoat-plugin/tree/master/example
The plugin is currently being processed by the Gradle team for publishing.
Please, don't take it as a self-advertising :)

**Comment by eugene-sy** [comment]
The plugin was published some time ago: https://plugins.gradle.org/plugin/com.github.eugenesy.scapegoat

**Comment by jozic** [comment]
seems like this should be closed now
